dask/array/routines.py

```diff
@@ -9,6 +9,7 @@
 from operator import getitem
 
 import numpy as np
+from six import string_types
```

Comment:
We actually don't depend on six.  I recommend just using `str` instead.  If you need something cross version then use `from .compatibility import str`